### PR TITLE
Sdk 2.6.0 integration

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

**Comment by wtrocki** [comment]
Ping @danielpassos @StephenCoady Going to do rounds on this to see how this will work on native clients and web.

File: server/src/tasks/schema.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 const { pubSub } = require('../subscriptions')
-const { conflictHandler } = require("@aerogear/voyager-conflicts")
+const { conflictHandler } = require("offix-conflicts-server")
```
Comment (wtrocki) [inline server/src/tasks/schema.js]: Waiting for the release of the voyager-server
CC: @darahayes

Comment (darahayes) [reply server/src/tasks/schema.js]: Working on that today

Comment (wtrocki) [reply server/src/tasks/schema.js]: Right. Let me know once you have any release. I want to make sure we are testing the right package.

**Pull request merged**